### PR TITLE
Create ui-c8y-1019-23-0-toogle-switch-for-password-case-sensitive-5334-graft-release-cd-6003.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by c8y-ui-automations[bot]

Ticket: [MTM-56865](https://cumulocity.atlassian.net/browse/MTM-56865)
Original PR: [6003](https://github.softwareag.com/IOTA/cumulocity-ui/pull/6003)

### CODE REVIEW

File: content/change-logs/platform-services/ui-c8y-1019-23-0-toogle-switch-for-password-case-sensitive-5334-graft-release-cd-6003.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: toogle switch for password case sensitive (#5334) [GRAFT][release/cd] (#6003)
```
Comment (gen-App-Enablement) [inline content/change-logs/platform-services/ui-c8y-1019-23-0-toogle-switch-for-password-case-sensitive-5334-graft-release-cd-6003.md:3]: The Claude AI suggested to use the following title:
```suggestion
title: Add toggle switch for case-sensitive passwords
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

**Review by pink-mongoose — APPROVED** [review]

**Review by pink-mongoose — COMMENTED** [review]
To enhance security options, a toggle switch has been added to the password settings which enables case-sensitive passwords. Previously, passwords were not case-sensitive. With this change, when the toggle is enabled, the password check will consider upper and lower case letters as distinct, requiring users to enter their password exactly as set. This impacts all users setting new passwords or changing existing ones if the toggle is enabled by an administrator.

**Comment by BeateRixen** [comment]
@kalavathig1 @sarangovindarajan Please implement AI and Jane´s suggestions and approve

**Review by mmed-softwareag — APPROVED** [review]

File: content/change-logs/platform-services/ui-c8y-1019-23-0-toogle-switch-for-password-case-sensitive-5334-graft-release-cd-6003.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: q3kclF6pO
+    label: Authentication
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-56865
+version: 1019.23.0
+---
+To enhance the user experience, a switch has been added to the authentication settings, allowing the option to disable case sensitivity in usernames. With this change, when the switch is turned on, username validation will treat uppercase and lowercase letters as distinct, requiring users to enter their username exactly as it was set. This affects all users.
```
Comment (BeateRixen) [inline content/change-logs/platform-services/ui-c8y-1019-23-0-toogle-switch-for-password-case-sensitive-5334-graft-release-cd-6003.md:17]:
```suggestion
To enhance the user experience, a switch has been added to the authentication settings, which allows to disable/enable case-sensitivity in usernames. When the switch is turned on, username validation will now treat uppercase and lowercase letters as distinct, requiring users to enter their username exactly as it was set. This affects all users.
```

File: content/change-logs/platform-services/ui-c8y-1019-23-0-toogle-switch-for-password-case-sensitive-5334-graft-release-cd-6003.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: A switch enabling the differentiation of letter case in user names.
```
Comment (BeateRixen) [inline content/change-logs/platform-services/ui-c8y-1019-23-0-toogle-switch-for-password-case-sensitive-5334-graft-release-cd-6003.md:3]:
```suggestion
title: New switch for disabling/enabling case-sensitivity in usernames
```

**Review by pink-mongoose — APPROVED** [review]

**Review by pink-mongoose — APPROVED** [review]

**Review by kalavathig1 — APPROVED** [review]

**Pull request merged**